Android/ZensoftHRCRM/app/src/main/java/com/erkprog/zensofthrcrm/data/db/SQLiteHelper.java

```diff
@@ -145,7 +145,7 @@
       ID + " INTEGER_PRIMARY_KEY, " +
       FIRST_NAME + " TEXT, " +
       LAST_NAME + " TEXT, " +
```

Comment:
Why you havent used room db framework)

Reply:
I've just found out about that. Will know -_-.